tests/test_torchmodule.py

```diff
@@ -107,6 +107,7 @@ def test_TorchSineVCO(self):
         )
 
     def test_TorchFmVCO(self):
+        # Is this correct?
         numpymod = numpymodule.FmVCO()
         torchmod = torchmodule.TorchSineVCO()
```

Comment:
Not really part of this PR, but wanted to flag this. should this be comparing FmVCO and TorchFmVCO -- why is this test working?

Reply:
hmmm @maxsolomonhenry ? thoughts?

Reply:
Hah. Whoops. Would you mind tweaking it? Odd how it didn't seem to trigger any failure...

Reply:
Consider it tweaked. That is odd that it didn't trigger any failure?